ci/jjb/scripts/pulp-backup.sh

```diff
@@ -0,0 +1,14 @@
+sudo yum -y install ansible attr git libselinux-python
+echo 'localhost' > hosts
+source "${RHN_CREDENTIALS}"
+export ANSIBLE_CONFIG="${PWD}/ci/ansible/ansible.cfg"
```

Comment:
The lines 1-4 are being repeated, maybe we could extract them to a separated file and include it whenever we want to run an ansible playbook.

Also we should the hosts definition outside that file. The hosts, specially when using localhost, can be passed directly to the `ansible-playbook` command like `ansible-playbook [other options] -i "localhost,"`. The trailing comma is required otherwise ansible will think it is a file and not a list of hosts being passed directly.